client/src/components/Home/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from 'antd';
+import './style.css';
+
+function Home() {
+  return (
+    <section className='home'>
+      <h1 className='home--heading'>
+        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. 
+        Aenean commodo
+      </h1>
+      <div className='home--buttons-container'>
+        <Button block className='home--button'>Login</Button>
```

Comment:
where the handlers for those buttons ? 
you should redirect to Login with login button and to create account with create profile button .
```suggestion
        <Button block className='home--button'>Login</Button>
```

Reply:
Oh sorry, I forget this :grin: